ush/ioda/bufr2ioda/bufr2ioda_ncep_satwind_amv_goes.py

```diff
@@ -0,0 +1,530 @@
+    q.add('expectedError', '*/AMVQIC{4}/PCCF')
+
+#   # Derived Motion Wind (DMW) Intermediate Vectors - Coefficient of Variation
+#   q.add('coefficientOfVariation', '*/AMVIVR/CVWD')
+    q.add('coefficientOfVariation', '*/AMVIVR{1}/CVWD')
+
+    # Wind Retrieval Method Information
+    q.add('windComputationMethod', '*/SWCM')
+    q.add('windHeightAssignMethod', '*/EHAM')
+
+    # ObsValue
+    q.add('windDirection', '*/WDIR')
+    q.add('windSpeed', '*/WSPD')
+
+    end_time = time.time()
+    running_time = end_time - start_time
+    print('Running time for making QuerySet : ', running_time, 'seconds')
+
+    # ==============================================================
+    # Open the BUFR file and execute the QuerySet to get ResultSet
+    # Use the ResultSet returned to get numpy arrays of the data
+    # ==============================================================
+    start_time = time.time()
+
+    print('Executing QuerySet to get ResultSet ...')
+    with bufr.File(DATA_PATH) as f:
+        r = f.execute(q)
+
+    print(' ... Executing QuerySet: get metadata: basic ...')
+    # MetaData
+    satid = r.get('satelliteId')
+    year = r.get('year')
+    month = r.get('month')
+    day = r.get('day')
+    hour = r.get('hour')
+    minute = r.get('minute')
+    second = r.get('second')
+    lat = r.get('latitude')
+    lon = r.get('longitude')
+    satzenang = r.get('sensorZenithAngle')
+    pressure = r.get('pressure', type='float')
+    chanfreq = r.get('sensorCentralFrequency', type='float')
+
+    print(' ... Executing QuerySet: get metadata: processing center ...')
+    # Processing Center
+    ogce = r.get('dataProviderOrigin')
+
+    print(' ... Executing QuerySet: get metadata: data quality information ...')
+    # Quality Information
+    qifn = r.get('qualityInformationWithoutForecast', type='float')
+    ee = r.get('expectedError', type='float')
+
+    print(' ... Executing QuerySet: get metadata: intermediate vector information ...')
+    # Derived Motion Wind (DMW) Intermediate Vectors
+    cvwd = r.get('coefficientOfVariation')
+
+    print(' ... Executing QuerySet: get metadata: wind retrieval method ...')
+    # Wind Retrieval Method Information
+    swcm = r.get('windComputationMethod')
+    eham = r.get('windHeightAssignMethod')
+
+    print(' ... Executing QuerySet: get obsvalue: wind direction and speed ...')
+    # ObsValue
+    # Wind direction and Speed
+    wdir = r.get('windDirection', type='float')
+    wspd = r.get('windSpeed')
+
+    print(' ... Executing QuerySet: get datatime: observation time ...')
+    # DateTime: seconds since Epoch time
+    # IODA has no support for numpy datetime arrays dtype=datetime64[s]
+    timestamp = r.get_datetime('year', 'month', 'day', 'hour', 'minute', 'second').astype(np.int64)
+
+    print(' ... Executing QuerySet: Done!')
+
+    print(' ... Executing QuerySet: Check BUFR variable generic dimension and type ...')
```

Comment:
We should try to use the `wxflow` logger and make some of this stuff "debug" and some "info"